[docs] Document the `browser-init-timeout` feature (#5878)
DevExpress/testcafe: docs/articles/documentation/reference/testcafe-api/runner/run.md

async run(options) → Promise<Number>
```

> Before TestCafe runs tests, it reads settings from the `.testcaferc.json` [configuration file](../../configuration-file.md) if this file exists. Then it applies settings specified in the programming API. API settings override values from the configuration file if they differ. TestCafe prints information about every overridden property in the console.

Reviewer: ```suggestion
Before TestCafe runs tests, it reads settings from the `.testcaferc.json` [configuration file](../../configuration-file.md) if this file exists and applies settings specified in the programming API. API settings override values from the configuration file. TestCafe prints information about every overridden property in the console.
```

> > Important! Make sure to keep the browser tab that runs tests active. Do not minimize the browser window.

Reviewer: ```suggestion
> Important! Make sure the browser tab that runs tests stays active. Do not minimize the browser window.
```

If a browser stops responding while it executes tests, TestCafe restarts the browser and reruns the current test in a new browser instance.
If the same problem occurs with this test two more times, the test run finishes and an error is thrown.

> > When you use a [LiveModeRunner](../livemoderunner.md), you can call the `runner.run` method only once. In rare cases when you need multiple live mode sessions to run in parallel, you can create multiple [TestCafe server instances](../testcafe/README.md).

Reviewer: ```suggestion
> When you use a [LiveModeRunner](../livemoderunner.md), can call the `runner.run` method only once. If you want to run multiple live mode sessions simultaneously, you can create multiple [TestCafe server instances](../testcafe/README.md).
```

Author: Can we simplify:

> When you use a LiveModeRunner, call the `runner.run` method **once**. To run multiple live mode sessions simultaneously, create multiple TestCafe server instances.

Reviewer: Ok

> > When you use a [LiveModeRunner](../livemoderunner.md), you can call the `runner.run` method only once. In rare cases when you need multiple live mode sessions to run in parallel, you can create multiple [TestCafe server instances](../testcafe/README.md).

Author: ```suggestion
> When you use a [LiveModeRunner](../livemoderunner.md), call the `runner.run` method once. To run multiple live mode sessions simultaneously, create multiple [TestCafe server instances](../testcafe/README.md).
```
